### PR TITLE
Format readme

### PR DESCRIPTION
**Pull request by furtnerthomas**
format README.md with markdown lint

### CODE REVIEW

**Comment by soulitzer** [comment]
Thanks for the change. Unfortunately this also seems to alter the style of the markdown.

**Review by soulitzer — CHANGES_REQUESTED** [review]
.

File: README.md [hunk 1]
```diff
@@ -348,14 +360,16 @@ You can adjust the configuration of cmake variables optionally (without building
 the following. For example, adjusting the pre-detected directories for CuDNN or BLAS can be done
 with such a step.
 
-On Linux
+**Linux**
```
Comment (furtnerthomas) [inline README.md:363]: having ** instead of unordered list would be more consistent

Comment (furtnerthomas) [inline README.md:363]: having "**" instead of "-" would be more consistent

**Pull request closed without merging**